code/modules/hydroponics/seeds.dm

```diff
@@ -223,7 +223,7 @@
 		CRASH("[T] has no reagents.")
 
 	for(var/rid in reagents_add)
-		var/amount = max(1, round(potency * reagents_add[rid], 1)) //the plant will always have at least 1u of each of the reagents in its reagent production traits
+		var/amount = max(1, round((potency * reagents_add[rid])/2, 1)) //the plant will always have at least 1u of each of the reagents in its reagent production traits

```

Comment:
holy shit this is the laziest fix ever

you just effectively cut the amount of chems made by each plant chem trait in half and called it a day lmao

Reply:
Shouldn't this be foo/length(reagents_iadd) instead of foo/2?

Reply:
And even that's not right because it assumes each reagent has an equal share.